specifications/api/swagger.yaml

```diff
@@ -2228,69 +2227,6 @@ components:
-            by an era-specific function whose value increases:
-
-              - as the number of different assets increases;
-              - as the quantity of any individual asset increases;
-              - as the length of the target address increases; and
-              - if a datum hash is added.
-
-            Therefore, the value reported by this field should only be viewed
-            as an absolute minimum, and only applies to outputs that send ada
-            (and no other assets) to addresses of the minimum possible length.
-            (At the time of writing, this corresponds to a 29-byte long
-            Shelley-era address.)
-
-            In practice, if an output contains other assets, specifies a datum
-            hash, or sends funds to an address that is longer than the absolute
-            minimum length, then the minimum value required by the ledger (and
-            the wallet) will be higher than the value reported by this field.
-
-            When using the wallet to construct or balance a transaction:
-
-              - if the caller specifies an output without an ada quantity, then
-                the wallet will automatically assign a minimal ada quantity to
-                the output that is guaranteed to be accepted by the ledger;
-
-              - if the caller specifes an output with a non-zero ada quantity,
-                then the wallet will verify that the specified quantity is not
-                less than the minimum quantity required by the ledger, and if
-                this verification step fails, return an error that reports the
-                required minimum.
-
-            In the Shelley, Allegra, and Mary eras, the `minimum_utxo_value`
-            field was equivalent to the ledger `minUTxOValue` protocol
-            parameter.
-
-            In the Alonzo era, the `minUTxOValue` protocol parameter was
-            replaced by the `coinsPerUTxOWord` protocol parameter. In this era,
-            the minimum ada quantity for an output was determined by
-            multiplying the `coinsPerUTxOWord` parameter by the length (in
-            8-byte words) of the in-memory representation of that output, which
-            was not dependent on the length of the address. Therefore, in this
-            era, specifying a longer address would not require an increase in
-            the minimum ada quantity.
-
-            In the Babbage era, the `coinsPerUTxOWord` protocol parameter was
-            replaced by the `coinsPerUTxOByte` protocol parameter. In this era,
-            the minimum ada quantity for an output is determined by multiplying
-            the `coinsPerUTxOByte` parameter by the length (in bytes) of the
-            serialised representation of the output, which is dependent on the
-            length of the address (among other factors).  Therefore, in this
-            era, specifying a longer address will require an increase in the
-            minimum ada quantity.
```

Comment:
In a subsequent PR, I will re-add parts of this description to the descriptions of other endpoints. In particular, I aim to ensure that all endpoints involving transaction construction have suitable documentation of the following behaviour:
- If the user specifies a transaction output `out` without any ada (or an ada quantity of zero), the wallet will automatically assign a minimal quantity of ada that satisfies the ledger rules.
- If the user specifies a transaction output `out` with an ada quantity `q`, the wallet will verify that `q` is large enough to satisfy the ledger minimum UTxO rule, and if `q` is too low, return an error with a revised quantity `r. r > q` that does satisfy the ledger minimum UTxO rule. The onus is then on the user to resubmit their request with the revised quantity `r` (or some value that is greater than `r`).